NEWS.md

```diff
@@ -1,10 +1,14 @@
-# tern.mmrm 0.3.2.9006
+# tern.mmrm 0.3.3
+
+### Miscellaneous
+
+* Created additional tests for plot functions.
```

Comment:
addied `parent_name` to ensure a unique row name, due to upstream rtables enhancement.

Reply:
NEWS updated.